lib/pbench/agent/tool_meister.py

```diff
@@ -79,7 +80,7 @@
 fmtstr = "%(asctime)s %(levelname)s %(process)s %(thread)s %(name)s %(funcName)s %(lineno)d -- %(message)s"
 
 
-def log_subprocess_output(pipe: subprocess.PIPE, logger: logging.Logger):
+def log_pipe_output(pipe: io.IOBase, logger: logging.Logger):
```

Comment:
I'm not going to comment that it's a bit odd that you changed the method name to use the word "pipe" in the same commit where you changed the _type_ of what you're logging from "pipe" to `io.IOBase`. 😁

Reply:
Thanks.  =)